src/components/MarkersDisplay/Annotations/AnnotationRow.js

```diff
@@ -67,6 +67,12 @@ const AnnotationRow = ({
   const handleOnClick = useCallback((e) => {
     e.preventDefault();
     checkCanvas();
+
+    // Handle click on a link in the text in the same tab without seeking the player
+    if (e.target.tagName == 'A') {
```

Comment:
Does this need an additional check that it isn't an anchor tag link or some other link that refers to the current page?